drivers/mmc/core/sdio_bus.c

```diff
@@ -273,6 +273,8 @@ static void sdio_release_func(struct device *dev)
 	 * cis tables for this func
 	 */
 	if (!func->card->host->embedded_sdio_data.funcs)
+#else
```

Comment:
I'm not sure about this `#else`: To me it looks like the `sdio_free_func_cis` function should not be called for embedded SDIO **or** non-standard SDIO. We don't seem to use `CONFIG_MMC_EMBEDDED_SDIO` so it shouldn't matter in practice but I think this `#else` needs to be the `#endif` from https://github.com/whatawurst/android_kernel_sony_msm8998/pull/82/files#diff-0b4db9467c20a99828df56af64d17a2fa58aa1ed240ef133078480a15283437eR278